libs/core/langchain_core/runnables/graph_mermaid.py

```diff
@@ -164,7 +164,15 @@ def draw_mermaid_png(
     Raises:
         ValueError: If an invalid draw method is provided.
     """
-    if draw_method == MermaidDrawMethod.PYPPETEER:
+    if draw_method == MermaidDrawMethod.PLAYWRIGHT:
+        import asyncio
+
+        img_bytes = asyncio.run(
```

Comment:
This does not handle the event loop correctly, so we'll need to fix this prior to merging